### DISCUSSION TITLE
com_categories - saveOrderAjax() not working

### DISCUSSION THREAD

**Discussion by PStanislav** [joomla/joomla-cms#37821] [General] [answered]
Example of request parameters:
{ "order[]": [ "1","9","1","2"] }
{ "cid[]": ["2","8"] }

will never pass:
 \count($idArray) == \count($lftArray)

in libraries\src\Table\Nested.php 
saveorder($idArray = null, $lftArray = null)

**Comment by joomdonation** [comment, marked as answer]
Should be fixed by this PR https://github.com/joomla/joomla-cms/pull/37781 . You can use the solution here https://github.com/joomla/joomla-cms/issues/37778#issuecomment-1124537696 while waiting for 4.1.4 release
